### PR TITLE
Feat/alert dialog example

### PR DESCRIPTION
**Pull request by sandeepgeeky**
1 - Added AlertDialog Example 
2 - Resolve previous examples.
3- changes in. avatar group example

### CODE REVIEW

File: example/storybook/src/components/Disclosure/Actionsheet/index.stories.mdx [hunk 1]
```diff
@@ -609,9 +609,8 @@ function App(){
                     Confirm security code
                   </FormControl.Label.Text>
                 </FormControl.Label>
-                <Input w="$full">
-                 
-                  <Input.Input placeholder="CVC/CVV" />
+                <Input w="$full" mt={5}>
```
Comment (amars29) [inline example/storybook/src/components/Disclosure/Actionsheet/index.stories.mdx:612]: Generally we don't take fullWidth in Input

File: example/storybook/src/components/Forms/Checkbox/index.stories.mdx [hunk 2]
```diff
@@ -461,30 +461,26 @@ The Examples section provides visual representations of the different variants o
...
-<Checkbox justifyContent='left' size="lg">
-  <Checkbox.Indicator>
-    <Checkbox.Icon>
-      <Icon as={CheckIcon} />
-    </Checkbox.Icon>
-  </Checkbox.Indicator>
-  <Checkbox.Label>Adobe XD</Checkbox.Label>
-</Checkbox>
-</VStack>
+          <VStack space="3xl">
```
Comment (amars29) [inline example/storybook/src/components/Forms/Checkbox/index.stories.mdx:464]: please correct the indentation in this example

File: example/storybook/src/components/Overlay/AlertDialog/index.stories.mdx [hunk 3]
```diff
@@ -409,6 +420,82 @@ The following are the lists of all the libraries and packages the component reli
...
+                         <Button.Text>Click me</Button.Text>
+                      </Button>
+                  </Center>
+                  <AlertDialog
+                          isOpen={showAlertDialog}
+                          onClose={() => {
+                              setShowAlertDialog(false);
+                          }}
+                          bgColor="white">
+                    <AlertDialog.Content bgColor="white">
```
Comment (amars29) [inline example/storybook/src/components/Overlay/AlertDialog/index.stories.mdx:447]: Use bg instead bgColor if possible.

File: example/storybook/src/components/Overlay/AlertDialog/index.stories.mdx [hunk 4]
```diff
@@ -409,6 +420,82 @@ The following are the lists of all the libraries and packages the component reli
...
+                      <AlertDialog.Body p='$5'>
+                        <HStack space="md" alignItems='center'>
+                          <AlertTriangle color="#B91C1C" size={16}/>
```
Comment (amars29) [inline example/storybook/src/components/Overlay/AlertDialog/index.stories.mdx:450]: Add a comment that you are importing external icons from lucid-icons.

Comment (sandeepgeeky) [reply example/storybook/src/components/Overlay/AlertDialog/index.stories.mdx:450]: done

File: example/storybook/src/components/Disclosure/Actionsheet/index.stories.mdx (hunk 1 above)
Comment (sandeepgeeky) [reply example/storybook/src/components/Disclosure/Actionsheet/index.stories.mdx:612]: okay

File: example/storybook/src/components/Forms/Checkbox/index.stories.mdx (hunk 2 above)
Comment (sandeepgeeky) [reply example/storybook/src/components/Forms/Checkbox/index.stories.mdx:464]: done

File: example/storybook/src/components/Disclosure/Actionsheet/index.stories.mdx [hunk 5]
```diff
@@ -609,9 +609,8 @@ function App(){
                     Confirm security code
                   </FormControl.Label.Text>
                 </FormControl.Label>
-                <Input w="$full">
-                 
-                  <Input.Input placeholder="CVC/CVV" />
+                <Input  mt={5}>
```
Comment (amars29) [inline example/storybook/src/components/Disclosure/Actionsheet/index.stories.mdx:612]: correct space

File: example/storybook/src/components/Feedback/Spinner/index.stories.mdx [hunk 6]
```diff
@@ -127,9 +125,9 @@ The Examples section provides visual representations of the different variants o
     metaData={{
       code: `
         <VStack space="md" >
-      <Spinner size="small"/>
-      <Spinner size="large"/>
-      </VStack>
+            <Spinner size="small"/>
```
Comment (amars29) [inline example/storybook/src/components/Feedback/Spinner/index.stories.mdx:128]: correct indentation.

File: example/storybook/src/components/Feedback/Spinner/index.stories.mdx [hunk 7]
```diff
@@ -153,10 +151,10 @@ The Examples section provides visual representations of the different variants o
     metaData={{
       code: `
         <VStack space="md" >
-      <Spinner color="#4F46E5" size="large"/>
-      <Spinner  color="#059669" size="large"/>
-       <Spinner  color="#D97706" size="large"/>
-      <Spinner  color="#C026D3" size="large"/>
+      <Spinner color="$indigo600" size="large"/>
```
Comment (amars29) [inline example/storybook/src/components/Feedback/Spinner/index.stories.mdx:154]: correct indentation.

File: example/storybook/src/components/Forms/Checkbox/index.stories.mdx [hunk 8]
```diff
@@ -412,16 +414,14 @@ The Examples section provides visual representations of the different variants o
...
-  <Checkbox.Indicator>
-    <Checkbox.Icon>
-      <Icon as={CheckIcon} />
-    </Checkbox.Icon>
-  </Checkbox.Indicator>
-  <Checkbox.Label>Label</Checkbox.Label>
-</Checkbox>
-</Center>
+        <Center>
+        <Checkbox {...props} >
```
Comment (amars29) [inline example/storybook/src/components/Forms/Checkbox/index.stories.mdx:418]: correct indentation.

File: example/storybook/src/components/Forms/Checkbox/index.stories.mdx [hunk 9]
```diff
@@ -461,30 +461,26 @@ The Examples section provides visual representations of the different variants o
...
-    <Checkbox.Icon>
-      <Icon as={CheckIcon} />
-    </Checkbox.Icon>
-  </Checkbox.Indicator>
-  <Checkbox.Label>Adobe XD</Checkbox.Label>
-</Checkbox>
-</VStack>
+          <VStack space="3xl">
+              <Checkbox defaultIsChecked size="lg">
+                      <Checkbox.Indicator bgColor="$darkBlue600" >
```
Comment (amars29) [inline example/storybook/src/components/Forms/Checkbox/index.stories.mdx:466]: correct indentation.

File: example/storybook/src/components/Forms/Checkbox/index.stories.mdx [hunk 10]
```diff
@@ -503,30 +499,26 @@ The Examples section provides visual representations of the different variants o
...
-    <Checkbox.Icon>
-        <Icon as={CheckIcon} />
-    </Checkbox.Icon>
-  </Checkbox.Indicator>
-   <Checkbox.Label>Typography</Checkbox.Label>
-</Checkbox>
-</HStack>
+          <HStack space="md">
+            <Checkbox size="lg">
+                <Checkbox.Indicator>
```
Comment (amars29) [inline example/storybook/src/components/Forms/Checkbox/index.stories.mdx:504]: correct indentation.

**Pull request merged**